pkg/plugins/runtime/k8s/util/util.go

```diff
@@ -143,3 +143,11 @@ func MeshFor(obj kube_meta.Object) string {
 
 	return mesh
 }
+
+func ServiceTagFor(svc *kube_core.Service, svcPort *kube_core.ServicePort) string {
```

Comment:
Would be nice to have godoc :)

Reply:
Maybe... the `godoc` comment would probably turn out something like `// ServiceTagFor returns the service tag for a service and port` which doesn't say _too_ much more than the signature.
Though I suppose I can mention `svcPort` being optional. If only the pointer type in `go` didn't mix the semantics of missing value and shared, mutable value.